test/HHConfiguration.py

```diff
@@ -13,17 +13,24 @@
         VarParsing.varType.int,
         'If running over MC (0) or data (1)')
 
+options.register('hltProcessName',
```

Comment:
This should not be needed, as it should be handled automatically by the framework. Do you have an error without?